ctapipe/reco/hillas_intersection.py

```diff
@@ -495,25 +497,19 @@ def reconstruct_xmax(
             np.array(ty),
         )
         weight = np.array(amp)
-        mean_height = np.sum(height * weight) / np.sum(weight)
+        mean_distance = np.sum(height * weight) / np.sum(weight)
```

Comment:
This is `np.average(height, weights=weight)`

Reply:
done